unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/BackpackEditorHUDController.cs

```diff
@@ -221,7 +221,7 @@ async UniTaskVoid LoadUserProfileAsync(UserProfile userProfile, CancellationToke
                     model.skinColor = userProfile.avatar.skinColor;
                     model.hairColor = userProfile.avatar.hairColor;
                     model.eyesColor = userProfile.avatar.eyeColor;
-
+                    model.forceRender = new HashSet<string>(userProfile.avatar.forceRender);
```

Comment:
Should we set a null value into the HashSet in case `forceRender` is null? maybe in that case we should asign an empty HashSet, right?